api/add_job.md

```diff
@@ -48,6 +50,16 @@ END
 $$;
 
 SELECT add_job('user_defined_action','1h');
+SELECT add_job('user_defined_action','1h', fixed_schedule => false);
 ```
 
+Register the `user_defined_action` procedure to run at midnight every Sunday:
+The `initial_start` provided must satisfy these requirements, so it must be a Sunday midnight.
+
+```sql
+-- December 4, 2022 is a Sunday
+SELECT add_job('user_defined_action','1 week', initial_start => '2022-12-04 00:00:00+00'::timestamptz);
+-- if subject to DST
+SELECT add_job('user_defined_action','1 week', initial_start => '2022-12-04 00:00:00+00'::timestamptz, timezone => 'Europe/Berlin');
```

Comment:
not sure why these examples are not displayed, so I need some pointers here on what else I should change